designs/umbra/src/base.mjs

```diff
@@ -0,0 +1,813 @@
+import { stretchToScale } from '@freesewing/core'
+import { formatPercentage } from '@freesewing/shared/utils.mjs'
```

Comment:
This does not seem like a valid import.

Reply:
Note that the `formatPercentage` method is a one-liner:

```mjs
const formatPercentage = (val) => Math.round(1000 * val) / 10 + '%'
```

You might just want to inline it.